src/main/java/org/humancellatlas/ingest/audit/AuditLogService.java

```diff
@@ -0,0 +1,23 @@
+package org.humancellatlas.ingest.audit;
+
+import lombok.RequiredArgsConstructor;
+import org.humancellatlas.ingest.core.AbstractEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+@Service
+@RequiredArgsConstructor
+public class AuditLogService {
+
+    private final AuditLogRepository auditLogRepository;
+
+    public void addAuditLog(AuditLog auditLog) {
+        auditLogRepository.save(auditLog);
+    }
+
+    public List<AuditLog> getAuditLogOf(AbstractEntity entity) {
```

Comment:
I would call this `getAuditLogForEntity`

Reply:
yea, that's the kind  of naming I was trying to go for...

If it doesn't read well with the team, I can change it

Reply:
I disagree. I think the naming should help the reader understand the purpose of the method.
Although it is correct that `getAuditLogOf` is like the way one would speak, it is not the purpose of the method's name.
The name `getAuditLogOf` does not help the reader understand what the parameter is or should be. I believe it helps to have a name like `getAuditLogForEntity`. This is my recommendation. You do not have to take it now.

Reply:
renamed to match @amnonkhen's suggestion